ztp/gitops-subscriptions/argocd/README.md

```diff
@@ -61,7 +118,7 @@ In order to deploy the OpenShift GitOps operator v1.12 you may apply the provide
   - Update *URL* to point to git repository. The URL must end with .git, eg: https://repo.example.com/repo.git
   - The *targetRevision* should indicate which branch to monitor
   - The path should specify the path to the directories holding SiteConfig or PolicyGenTemplate CRs respectively
-5. Apply pipeline configuration to your hub cluster using the following command.
+1. Apply pipeline configuration to your hub cluster using the following command.
```

Comment:
wrong numbering. Please compile the doc in a editor to avoid cosmetic issues.

Reply:
Fixed. I fixed these 2 numbering issues before (reported by Irina) The editor using visual code is not working right in this instance. Looks like it keep reverting to starting at 1. Hope this is correct now.